### ISSUE TITLE
Security TaskForce related issues

### ISSUE THREAD

**Issue by danielpeintner** [w3c/wot-scripting-api#315]
let's capture issues and topics related to Security task-force.

- [ ] Provision security
  -  [ ] https://github.com/w3c/wot-scripting-api/issues/299
  -  [ ] https://github.com/w3c/wot-scripting-api/issues/298

**Comment by relu91** [comment]
Possibly we could also discuss about how `node-wot` implements credential storage and security configuration. A while ago the security task force asked me to provide a testing playground to asses` node-wot` features. I created one [here](https://github.com/relu91/wot-security-testing-). What do you think @danielpeintner ?

**Comment by danielpeintner** [comment]
I agree. It makes perfect sense to experiment in code and have a playground! Thanks @relu91

**Comment by mmccool** [comment]
I labelled all the issues I could see related to security and added a link to those above.

**Comment by mmccool** [comment]
so... should not be using "security-tracker" label as it has a special meaning, cannot be removed by us, etc.  So I will create a new label, "Security", that we can use instead to bring items to the attention of the Security TF, and I will label all existing security-related issues with it.  Please use this label in the future...

**Comment by danielpeintner** [comment]
Note: there have been 3 more issues labeled with "security-tracker" which I removed now.
